src/protagonist/Engine/Settings/EngineSettings.cs

```diff
@@ -114,6 +114,15 @@ public class ImageIngestSettings
     /// </summary>
     public List<string> DefaultThumbs { get; set; } = new();
 
+    /// <summary>
+    /// A set of cookie names used by the load balancer to indicate stickiness
+    /// </summary>
+    public List<string> LoadBalancerStickinessCookieNames { get; set; } = new()

```

Comment:
Are these prefixes rather than set cookie names? To handle ALB cookies can split like `AWSALBAPP-*`

Reply:
these are the `key` value of a kvp, and are what I could see for an `lb_cookie` type of stickiness - the `AWSALPAPP-*` are   specific to `app_cookie` stickiness, and the reason they work like this, is (I believe) because the standard size of a cookie is 4k, and the load balancer `app_cookie` requires up to 16k, so it can split the cookie into a maximum of 4 shards.  This doesn't seem to be required by `lb_cookie` and the values below this line are what I could see being returned by the load balancer

Reply:
for context, this is what the cookie looks like coming back from the load balancer - `AWSALB=IOCtmCsAoVQyTEz3uphH7JfMhKI5/378EAs5ZyUSKDY1Uzpxt9kAfmR7yi3m5C17EKkSt0XVh08Zich3qUk6p7+eBxBR4ET/Xvzf61AbEPxruWKqg/zflioL8fQJ; Expires=Tue, 09 Jul 2024 15:45:02 GMT; Path=/`